src/Sylius/Bundle/CoreBundle/Resources/config/services/form.xml

```diff
@@ -52,6 +52,8 @@
             <parameter>sylius</parameter>
         </parameter>
         <parameter key="sylius.model.shop_billing_data.class">Sylius\Component\Core\Model\ShopBillingData</parameter>
+        <parameter key="sylius.catalog_promotion.action.fixed_discount" type="constant">Sylius\Bundle\CoreBundle\Calculator\FixedDiscountPriceCalculator::TYPE</parameter>
```

Comment:
I'm wondering if we could also remove these parameters, but I don't have any clear idea how for now

Reply:
To be honest, I would be for not using these parameters, because there are used only once, for tag in form definition, IMO there could be explicitly defined the proper type.